### PR TITLE
Niad 2126 report structured fhir messages scenario 1

### PR DESCRIPTION
**Pull request by benhession**
## ACCEPTANCE CRITERIA

**GIVEN** a test user wants to test a Patient referred to Primary Care
**THEN** opening the test area the following form fields should be visible…

ODS Code - Id: @@ods-code@@, NOT NULL

DOS Code - Id: @@dos-code@@

NHS Number- Id: @@nhs-number@@ NOT NULL

Recipient Name: @@recipent-name@@ NOT NULL

Recipient Address Line 1: @@recipent-street-address-line-1@@ NOT NULL

Recipient Address Line 2: @@recipent-street-address-line-2@@ NOT NULL

Recipient Town: @@recipent-street-address-town@@ 

Recipient City: @@recipent-street-address-city@@ 

Recipient Postcode: @@recipent-street-address-postcode@@ 

Encounter Start Datetime: @@nhs111-encounter-date-time-start@@, NOT NULL

Encounter End Datetime: @@nhs111-encounter-date-time-end@@, NOT NULL

Reported Conditions: @@patients_reported_condtion@@, NOT NULL

Pathways Disposition : @@Pathways Disposition@@, NOT NULL

Selected Service: @@selected_service@@, NOT NULL

Consultation Summary: @@consultation_summary@@, NOT NULL

Pathways Assessment: @@pathways-assessment@@, NOT NULL

Advice Given, @@advice_given@@, NOT NULL

Special Patient Notes, @@special_patient_notes@@, NOT NULL

**WHEN** entered and validated 
**THEN** the fields should be inserted into the example data given below
**AND** a request should be sent to the configured 111 Adapter endpoint.

### CODE REVIEW

File: test-suite/src/data/schema.ts [hunk 1]
```diff
@@ -222,94 +222,95 @@ const schema: Schema = {
...
+            "This test will attempt to send notification of a safeguarding referral to Social Services.",
           template: require("./safeguarding-request.xml"),
           testSpecifications: {
             [RequestHeaderProps.Header]: [
               {
                 label: "url",
                 id: "url",
                 defaultValue: "http://localhost:8081/report",
-                validators: [isUrl(), isReportUrl()]
+                validators: [isUrl(), isReportUrl()],
```
Comment (SRAlexander) [inline test-suite/src/data/schema.ts:234]: the comma is not required at the end here.

File: test-suite/src/data/schema.ts [hunk 2]
```diff
@@ -222,94 +222,95 @@ const schema: Schema = {
...
               },
               {
                 label: "Content Type",
                 id: "content-type",
                 defaultValue: "application/xml",
-                validators: [notNull(), minLength(7)]
-              }
+                validators: [notNull(), minLength(7)],
```
Comment (SRAlexander) [inline test-suite/src/data/schema.ts:240]: The same for both of these, no comma is required on the last item

Comment (SRAlexander) [reply test-suite/src/data/schema.ts:240]: It looks like this is a reoccurring issue, please review the rest of the changes below.

**Review by SRAlexander — CHANGES_REQUESTED** [review]

File: test-suite/src/data/schema.ts (hunk 1 above)
Comment (benhession) [reply test-suite/src/data/schema.ts:234]: I was of the the same opinion, it's enforced by the prettier formatter.

Comment (benhession) [reply test-suite/src/data/schema.ts]: I've made changes to the prettier config as discussed.

File: test-suite/src/data/schema.ts (hunk 2 above)
Comment (benhession) [reply test-suite/src/data/schema.ts]: Made the changes.

**Review by SRAlexander — APPROVED** [review]

**Pull request merged**